src/app/sitemap.ts

```diff
@@ -0,0 +1,49 @@
+import { MetadataRoute } from 'next';
+const siteUrl = 'https://digest.club';
+
+export default function sitemap(): MetadataRoute.Sitemap {
+  return [
+    {
+      url: siteUrl,
+      lastModified: new Date(),
+      changeFrequency: 'yearly',
+      priority: 1,
+    },
+    {
+      url: `${siteUrl}/updates`,
+      lastModified: new Date(),
+      changeFrequency: 'weekly',
+      priority: 0.6,
+    },
+    {
+      url: `${siteUrl}/discover`,
+      lastModified: new Date(),
+      changeFrequency: 'always',
+      priority: 0.7,
+    },
+    {
+      url: `${siteUrl}/account`,
+      lastModified: new Date(),
+      changeFrequency: 'always',
+      priority: 0.7,
+    },
+    {
+      url: `${siteUrl}/auth/login`,
+      lastModified: new Date(),
+      changeFrequency: 'yearly',
+      priority: 0.7,
+    },
+    {
+      url: `${siteUrl}/unsubscribe`,
+      lastModified: new Date(),
+      changeFrequency: 'yearly',
+      priority: 0.7,
+    },
+    {
+      url: `${siteUrl}/teams/create`,
+      lastModified: new Date(),
+      changeFrequency: 'yearly',
+      priority: 0.7,
+    },
```

Comment:
We can also add dynamically public team pages and digests:
- https://digest.club/slug-team/digest*

Reply:
Done ✅